src/App.tsx

```diff
@@ -13,11 +14,6 @@ import AppConfig from './services/configservice';
 import { PLAYGROUND_UI_COLORS } from './theme';
 
 export interface AppProps {
-  /**
-   * withRouter, it specified, is the router to wrap the application with.
-   */
-  withRouter?: any;
-
```

Comment:
I looked at index.tsx and realized that this only ever has one value, so I got rid of the prop passing as unnecessary complexity.